### ISSUE TITLE
Android screen reader talkback read improper On and off when double tab on the segment button tab

### ISSUE THREAD

**Issue by kawayiboy** [ionic-team/ionic#15804]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
cli packages: (C:\Users\telu\AppData\Local\Yarn\Data\global\node_modules)

    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

global packages:

    cordova (Cordova CLI) : 6.5.0
    Gulp CLI              : CLI version 2.0.1 Local version 3.9.1

local packages:

    @ionic/app-scripts : 3.1.9
    Cordova Platforms  : android 6.1.1 ios 4.5.5 windows 5.0.0
    Ionic Framework    : ionic-angular 3.8.0

System:

    Node : v8.11.3
    npm  : 5.6.0
    OS   : Windows 10

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro
```

**Describe the Bug**
User Experience:

Screen reader user will get confuse if incorrect information about the state of 'Feed' and 'Approvals' toggle button is provided to the screen reader when user selects any one of the two toggle buttons.

Pre-Requisite: Microsoft Flow app should be installed from Hockey app.

Screen reader(Talkback) should be activated.

**Steps to Reproduce**
1. Open Microsoft Flow app(which contains the segment button) installed on device. 
2. Navigate on the Home screen('Activity' button at the button should be selected on home screen) and try to select 'Feed' and 'Approvals' toggle button.
3. Verify the state information provided by the screen reader when user selects 'Feed' and 'Approvals' toggle button.

**Related Code**
<ion-segment class="p2" [ngModel]="activeSegment">
            <ion-segment-button value="Feed" aria-label="Feed" tappable (ionSelect)="onSegmentButtonSelected($event)">
                <div>{{'FeedPage_Title' | translate}}</div>
            </ion-segment-button>
            <ion-segment-button value="Approvals" aria-label="Approvals" tappable (ionSelect)="onSegmentButtonSelected($event)">
                <div [ngClass]="{'add-badge': viewModel.displayApprovalBadge}">
                    {{'ApprovalCenter_Title' | translate}}
                </div>
            </ion-segment-button>
        </ion-segment>

**Expected Behavior**
Screen reader should provide correct state information when user selects 'Feed' and 'Approvals' toggle button and state information should be consistent for both toggle buttons.

**Additional Context**
video: https://drive.google.com/file/d/1RHR4mdGfOyo4OS5F-bKfNU4LomnnDNY4/view?usp=sharing
